modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -41,7 +41,7 @@ export default async function convertB3dmToI3sGeometry(
   const result = [];
   let nodesCounter = nodeId;
   let {materials = []} = tileContent.gltf;
-  if (!materials.length === 0) {
+  if (!materials?.length) {
```

Comment:
It would be great if we could rename all files from `.js` to `.ts`, so we can use types everywhere.